paymentsheet/src/main/java/com/stripe/android/paymentsheet/LinkHandler.kt

```diff
@@ -42,22 +37,12 @@ internal class LinkHandler @Inject constructor(
     sealed class ProcessingState {
         data object Ready : ProcessingState()
 
-        data object Launched : ProcessingState()
-
         data object Started : ProcessingState()
 
         data class PaymentDetailsCollected(
             val paymentSelection: PaymentSelection?
         ) : ProcessingState()
 
-        data class Error(val message: String?) : ProcessingState()
-
-        data object Cancelled : ProcessingState()
-
-        data class PaymentMethodCollected(val paymentMethod: PaymentMethod) : ProcessingState()
-
-        data class CompletedWithPaymentResult(val result: PaymentResult) : ProcessingState()
-
         data object CompleteWithoutLink : ProcessingState()
     }
```

Comment:
Big win! `ProcessingState` now only refers to Link inline signup. We should be able to get rid of it completely once Link inline signup is moved as well.

Reply:
I'm so pumped for this!!!!!